heim-runtime/Cargo.toml

```
@@ -13,10 +13,12 @@ license = "Apache-2.0 OR MIT"
 futures = { version = "~0.3", default-features = false, features = ["std"] }
 #futures-lite = "~1.11"
 futures-timer = "~3.0"
+once_cell = "1.3.1"
```

Latest version of `once_cell` is `1.5.2` and introducing outdated dependency seems to be incorrect. Let's change it to
```suggestion
once_cell = "~1.5"